src/Avalonia.Visuals/Rendering/SceneGraph/SceneLayers.cs

```diff
@@ -13,6 +13,7 @@ public class SceneLayers : IEnumerable<SceneLayer>
         private readonly IVisual _root;
         private readonly List<SceneLayer> _inner;
         private readonly Dictionary<IVisual, SceneLayer> _index;
+        public static bool AllowGpuOptimisedRects { get; set; }
```

Comment:
Is this property name accurate? Developers might think it should be always enabled.